README.md

```diff
@@ -808,5140 +794,16 @@ Load balancing scheduler algorithms (`lbmethod`) are listed [in mod_proxy_balanc
-
-Boolean.
-
-Default: `true`.
-
-##### `purge_vhost_dir`
-
-If the [`vhost_dir`][] parameter's value differs from the [`confd_dir`][] parameter's, this parameter determines whether Puppet removes any configurations inside `vhost_dir` that are _not_ managed by Puppet.
-
-Setting `purge_vhost_dir` to `false` is a stopgap measure to allow the apache module to coexist with existing or otherwise unmanaged configurations within `vhost_dir`.
-
-Boolean.
-
-Default: same as [`purge_configs`][].
-
-##### `rewrite_lock`
-
-Allows setting a custom location for a rewrite lock - considered best practice if using a RewriteMap of type prg in the [`rewrites`][] parameter of your virtual host. This parameter only applies to Apache version 2.2 or lower and is ignored on newer versions.
-
-Default: `undef`.
-
-##### `sendfile`
-
-Forces Apache to use the Linux kernel's `sendfile` support to serve static files, via the [`EnableSendfile`][] directive. Values: 'On', 'Off'.
-
-Default: 'On'.
-
-##### `serveradmin`
-
-Sets the Apache server administrator's contact information via Apache's [`ServerAdmin`][] directive.
-
-Default: 'root@localhost'.
-
-##### `servername`
-
-Sets the Apache server name via Apache's [`ServerName`][] directive.
-
-Setting to `false` will not set ServerName at all.
-
-Default: the 'fqdn' fact reported by [Facter][].
-
-##### `server_root`
-
-Sets the Apache server's root directory via Apache's [`ServerRoot`][] directive.
-
-Default: Depends on operating system.
-
-- **Debian**: `/etc/apache2`
-- **FreeBSD**: `/usr/local`
-- **Gentoo**: `/var/www`
-- **Red Hat**: `/etc/httpd`
-
-##### `server_signature`
-
-Configures a trailing footer line to display at the bottom of server-generated documents, such as error documents and output of certain [Apache modules][], via Apache's [`ServerSignature`][] directive. Values: 'Off', 'On'.
-
-Default: 'On'.
-
-##### `server_tokens`
-
-Controls how much information Apache sends to the browser about itself and the operating system, via Apache's [`ServerTokens`][] directive.
-
-Default: 'Prod'.
-
-##### `service_enable`
-
-Determines whether Puppet enables the Apache HTTPD service when the system is booted.
-
-Boolean.
-
-Default: `true`.
-
-##### `service_ensure`
-
-Determines whether Puppet should make sure the service is running. Values: `true` (or 'running'), `false` (or 'stopped').
+### Templates
 
-The `false` or 'stopped' values set the 'httpd' service resource's `ensure` parameter to `false`, which is useful when you want to let the service be managed by another application, such as Pacemaker.
+The Apache module relies heavily on templates to enable the [`apache::vhost`][] and [`apache::mod`][] defined types. These templates are built based on [Facter][] facts specific to your operating system. Unless explicitly called out, most templates are not meant for configuration.
```

Comment:
Suggest omitting "heavily".

Also I'm not sure if the second sentence is saying "These templates are built based on [Facter][] facts *that are* specific to your operating system" or "These templates are built based on [Facter][] facts *and are* specific to your operating system". Are the facts specific to the OS or is it the templates that are specific to the OS?

Reply:
I believe it is meant to say that the facts are specific to the OS, as there are not OS specific templates. I'll update it accordingly.